src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests.cs

```diff
@@ -7098,6 +7118,49 @@ public static int Main()
                 Diagnostic(ErrorCode.ERR_AttributeCantBeGeneric, "Gen2").WithArguments("Gen2<T>"));
         }
 
+        [Fact]
+        public void OpenGenericTypeInAttributeWithGenericAttributeFeature()
+        {
+            var source =
+@"
+class Gen<T> {}
+class Gen2<T> : System.Attribute {}
+class Gen3<T> : System.Attribute { Gen3(T parameter) { } }
+	
+[Gen]
+[Gen2]
+[Gen2()]
+[Gen2<U>]
```

Comment:
This is an interesting scenario. It would be good to test that a type parameter from an outer type can be used in the attribute:
```C#
class Outer<T>
{
   [Gen2<T>]
   class Inner { }
}
```

Reply:
This is also should give error the clr has limitation that generic attribute cant be an open type

Reply:
Lost a lot of fun here... Hope that CoreCLR could support it in the future :D